library/wazo_wizard.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+
+# Copyright: (c) 2019, Wazo Communication Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+
+from wazo_confd_client import Client as Confd
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: wazo_wizard
+
+short_description: initial configuration of a Wazo engine
+
+version_added: "2.4"
+
+description:
+    - "This is my longer description explaining my sample module"
+
+options:
+    hostname:
+        description:
+            - This is the hostname of the wazo engine host
+        required: false
+        default: localhost
+    language:
+        description:
+            - language to use (ex. en_US).
+        required: true
+    password:
+        description:
+            - password for the Wazo engine
+        required: true
+    engine_api_port:
+        description:
+            - password for the Wazo engine
+        required: false
+        default: 443
+    https:
+        description:
+            - use https to connect to the Wazo engine
+        required: false
+        default:true
+    prefix:
+        description:
+            - prefix to contact the configuration service
+        required: false
+        default: /api/confd
+    verify_certificate:
+        description:
+            - verify SSL certificates
+        required: false
+        default: true
+    provisioning_step:
+        description:
+            - enable the provisioning step
+        required: false
+        default: true
+    manage_services_step:
+        description:
+            - enable the manage service step
+        required: false
+        default: true
+    manage_hosts_file_step:
+        description:
+            - enable the manage hosts file step
+        required: false
+        default: true
+    manage_resolv_file_step:
+        description:
+            - enable the manage resolv file step
+        required: false
+        default: true
+    commonconf_step:
+        description:
+            - enable the commonconf step
+        required: false
+        default: true
+
+author:
+    - Wazo (@wazocommunity)
+'''
+
+EXAMPLES = '''
+# Configure a Wazo engine
+- name: Test with a message
+  wazo_wizard:
+    language: en_US
+    password: solidpass
+'''
+
+RETURN = '''
+:
+    description: 
+    type: str
+    returned: always
+'''
```

Comment:
Since we don't return anything, [the doc](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block) says this should read: `RETURN = r''' # '''`

Reply:
done